crates/table/src/table.rs

```diff
@@ -66,9 +65,11 @@ pub struct Table {
     /// Page manager and row layout grouped together, for `RowRef` purposes.
     inner: TableInner,
     /// Maps `RowHash -> [RowPointer]` where a [`RowPointer`] points into `pages`.
-    pointer_map: PointerMap,
+    /// If `None`, there is a unique index that implies the set semantic constraint.
```

Comment:
```suggestion
    /// The [`PointerMap`], which is effectively a specialized unique index on all the columns.
    ///
    /// In tables without any other unique constraints, the pointer map is used to enforce set semantics,
    /// i.e. to prevent duplicate rows.
    /// If `self.indexes` contains at least one unique index, duplicate rows are impossible regardless,
    /// so this will be `None`.
```